packages/api/src/command/job/patient/scheduler/start-jobs.ts

```diff
@@ -78,15 +79,21 @@ export async function startJobs({
   );
   if (runJobsErrors.length > 0) {
     const msg = "Failed to run some scheduled patient jobs";
+    const errors = runJobsErrors.map(error => ({
+      id: error.id,
+      cxId: error.cxId,
+      error: errorToString(error.error),
+    }));
+    log(
+      `${msg}. Cause: ${errors
+        .map(error => `\n${error.id} ${error.cxId} ${error.error}`)
```

Comment:
nit: `cxId` and `patientId` can go on the log prefix since its a param for the fn, right?